src/test/java/com/dawn/jat/illuminati/post/advice/PostExceptionAdviceTest.java

```diff
@@ -29,10 +31,5 @@ void exception_throwException_returnHttpPostSummaryNoContent() {
         ResponseEntity<ErrorResponse> rsp = advice.exceptionPostSummary(postSummaryException);
         assertEquals("Cannot find post summary", postSummaryException.getMessage());
```

Comment:
we dont need to actually test with postSummaryException here though, if we need to test for coverage, better create another unit test for it.

Reply:
This part belong to exception_throwException_returnHttpPostSummaryNoContent TC, not TC exception_throwException_returnHttpPostNotFound